lib/ohai/common/dmi.rb

```diff
@@ -107,15 +107,15 @@ def convenience_keys(dmi)
               next if field.to_s == 'record_id'
               translated = field.downcase.gsub(/[^a-z0-9]/, '_')
               if in_common.has_key?(translated)
-                in_common[translated] = nil unless in_common[translated] == value
+                in_common[translated] = nil unless in_common[translated] == value.strip
```

Comment:
I think you should pull `value.strip` up since it is being repeated multiple times